cpp/include/cuda_shared_MV_multiply.h

```diff
@@ -0,0 +1,46 @@
+// The MIT License (MIT)
```

Comment:
no capitalized letters are allowed in a file name, you can rename it cuda_matrix_vector_multiply_shared_memory.h

Reply:
Fixed.